Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx

```diff
@@ -197,6 +198,29 @@ bool vtkMRMLCameraWidget::CanProcessInteractionEvent(vtkMRMLInteractionEventData
     return this->CanProcessButtonClickEvent(eventData, distance2);
     }
 
+  // Need to handle this event here because we keep handle generic event (rotation, translation,...)
```

Comment:
Would it be possible to revisit the wording associated with this comment ?

Reply:
We refactor the code so it's not needed anymore. Thanks for the feedback !